src/url-utils.js

```diff
@@ -76,3 +77,16 @@ export function sanitizeCodeRepositoryUrl(url) {
   }
   return url
 }
+
+export function getIpfsCidFromUri(string) {
+  const ipfsCid = string.replace(/^ipfs:/, '')
+
+  if (isIPFS.cid(ipfsCid) || isIPFS.cidPath(ipfsCid)) {
+    return ipfsCid
+  }
+  return ''
+}
+
+export function getIpfsUrlfromUri(uri) {
+  return `${ipfsDefaultConf.gateway}/${getIpfsCidFromUri(uri)}`
+}
```

Comment:
Is this the right place for these? did I miss any existing utils that already serve this purpose?

Reply:
Yes, I think this is the right place! We haven't had to handle completely generic IPFS URIs before (we only had to load app-related information).